pwem/emlib/lib.py

```diff
@@ -0,0 +1,30 @@
+# **************************************************************************
+# *
+# * Authors:     J.M. de la Rosa Trevin (delarosatrevin@scilifelab.se) [1]
+# *
+# * [1] SciLifeLab, Stockholm University
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 3 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+try:
+    from xmippLib import *
+except:
+    from ._libNone import *
```

Comment:
I would add a print of the exception, because there are different types of errors and often people report installation issues with the ghost triggered. Then, it will be easier if we have some hint of what is failing.
```
try:
    from xmippLib import *
except Exception as exc:
    from ._libNone import *
    print(exc)
```

Reply:
I like the idea if the exception can give some hint about the problem.

Reply:
Yes, because it's different to get an import error saying 'module not found' (you should install XmippBinaries) or any internal error (we have seen problems when a compiled Xmipp for Ubuntu distros is used in a Centos and the opposite, or other errors regarding libraries colisions).